create-lightning-wallet.md

```diff
@@ -0,0 +1,46 @@
+# How to create a Bitcoin wallet
+
+Creating a Lightning wallet on BlueWallet is easy.
+
+You can either click on the "Add now" button or the "+" icon in the top right corner
+
+![](./images/bluewallet-first-page.png)
+
+That will take you to the "add wallet" screen.
+
+## Adding a new wallet
+
+In the "add wallet" screen, you can choose between a Bitcoin wallet and a Lightning wallet.
+
+![](./images/add-lightning-wallet.png)
+
+For Lightning, click on the Lightning button. (If you want to create a Bitcoin wallet, see [this guide](./create-bitcoin-wallet.md).)
+
+You may also name your wallet, or leave the default name.
+
+<div class="side-note">
+
+### Advanced options
+
+If you have "Advanced mode" enabled in general settings,
+you'll see one more option for creating the wallet.
+
+![](./images/add-lightning-wallet-with-advanced-options.png)
+
+#### Connecting to your own LNDHub
+
+Lightning functionality in BlueWallet is powered by [LNDHub](http://lndhub.io/), an open source service that we created for this purpose. We host our own instance of this service and BlueWallet connects to it by default. However, for more privacy you can host your own instance and connect to it.
+
+To do that, you can add the address of your node in the specified text field.
+
+#### Provide entropy
+
+Is the "Provide entropy" feature applicable to Lightning wallets???
```

Comment:
Not sure if the "provide entropy" feature is applicable to Lightning wallets, is it?